torch/csrc/generic/StorageSharing.cpp

```diff
@@ -101,10 +101,10 @@ static PyObject * THPStorage_(shareFilename)(THPStorage *self)
     ctx = (libshm_context*)allocator_obj->allocatorContext;
   } else {
     // TODO: retry on collision
-    AutoNoGIL no_gil;
+    // TODO: free GIL - but remember to reacquire it when an exception is thrown
     THWStoragePtr new_storage(THPStorage_(newFilenameStorage)(storage->size));
-    THWStorage_(copy)(new_storage, storage);
```

Comment:
looks incorrect, prob revert changes to this file.

Reply:
Actually, I think this was https://github.com/pytorch/pytorch/pull/8887/commits/0ee126778f388a2fb13a115a604d68c51f89c54f and an actual failure internally. Do you still want to skip these changes?

Reply:
the revert commit is wrong. It makes `THWStoragePtr` into `THStoragePtr` (which will result in a compile error, as you see)